lambdas/fetch-questions/src/services/questions-retrieval-service.ts

```diff
@@ -0,0 +1,202 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { MetricUnits } from "@aws-lambda-powertools/metrics";
+
+import {
+  HTTPMetric,
+  ResponseValidity,
+} from "../../../../lib/src/MetricTypes/http-service-metrics";
+
+import { Classification } from "../../../../lib/src/MetricTypes/metric-classifications";
+
+enum QuestionServiceMetrics {
+  ResponseQuestionKeyCount = "ResponseQuestionKeyCount",
+  MappedQuestionKeyCount = "MappedQuestionKeyCount",
+}
+
+import { QuestionsResult, Question } from "../types/questions-result-types";
+
+const logger = new Logger({ serviceName: "QuestionsRetrievalService" });
+import { MetricsProbe } from "../../../../lib/src/Service/metrics-probe";
```

Comment:
Can we keep the imports above the const declarations

Reply:
Done, but we need an automatic way to catch this via the linter.